src/components/index/page-sections/StartupSection.jsx

```diff
@@ -0,0 +1,70 @@
+import { Link } from '@reach/router';
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PlanetImage from '../../../images/planet.png';
+// import SectionHeading from '../section-heading/SectionHeading';
+
+const NewsSection = () => {
+  return (
+    // Section Content Container
+    <div className="container mx-auto mb-40">
+      {/* Section Header */}
+      <h1 className="text-5xl mb-12">
+        <FormattedMessage id="index.startups.title" />
+      </h1>
+      {/* 2 ROW GRID - FOR 2 NEWS ARTICLES  */}
+      <div className="grid grid-cols-2 grid-flow-row gap-10">
+        {/* 2 COLS GRID - SEPARATING PHOTO AND TEXT */}
+        <div className="">
+          {/* EACH CONTENT CARD CAN BE OUTSOURCED INTO SINGLE CONTENT CARD COMPONENT */}
+          <div className="w-full h-auto items-center md:flex-col rounded-2xl gap-10">
+            <p className="text-left mb-20">
+              <FormattedMessage id="index.startups.description" />
+            </p>
+            <div className="w-full h-auto items-center md:flex-col mx-auto rounded-2xl -ml-32">
+              {/* FIRST RPW */}
+              <img src={PlanetImage} alt="" className="object-scale-down" />
+            </div>
+            {/* SECOND ROW */}
+          </div>
+        </div>
+        <div className="grid-rows-2 flex flex-col gap-10">
+          {/* FIRST STARTUP */}
+          <div className="w-full h-auto items-center bg-gray-primary rounded-2xl">
+            <h1 className="text-xl m-5 text-left">
```

Comment:
das ist in der Hierarchie der Seite mindestens h2 

https://www.w3schools.com/html/html_headings.asp

>Headings Are Important
>Search engines use the headings to index the structure and content of your web pages.
>
> Users often skim a page by its headings. It is important to use headings to show the document structure.
> 
> `<h1>` headings should be used for main headings, followed by `<h2>` headings, then the less important `<h3>`, and so on.